ansi.go

```diff
@@ -41,6 +41,8 @@ const (
 	CyanBackground           = "\x1b[46m"
 	GrayBackground           = "\x1b[47m"
 	ResetBackground          = "\x1b[49m"
+	Bold                     = "\033[1m"
+	BoldOff                  = "\033[0m"
```

Comment:
This is the same as Reset, I think bold off is 22m not 0m

If possible, does it work with \x1b prefix? Would like to keep consistent with the others if possible :)

Reply:
Done.